src/engraving/libmscore/edit.cpp

```diff
@@ -1888,7 +1892,10 @@ void Score::cmdAddOttava(OttavaType type)
 
         Ottava* ottava = new Ottava(this);
         ottava->setOttavaType(type);
-
+        if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```

Comment:
same

Reply:
Code duplication, again, would make sense to me to keep inside Ottava::setOttavaType

Reply:
That duplication is not my fault though, and I don't plan to fix that, not for this issue.